4-projects/business_unit_1/development/outputs.tf

```diff
@@ -29,6 +29,11 @@ output "peering_project" {
   value       = module.peering_project.project_id
 }
 
+output "env_secrets_project" {
+  description = "Project sample peering project id."
```

Comment:
nit: update desc

Reply:
changed in 99ccedd64f286ace5b0c733bfb6ed2cb80c46d39